components/blitz/src/ome/services/blitz/repo/ManagedImportRequestI.java

```diff
@@ -661,7 +661,10 @@ public MessageDigest parseData(
                 store.setChannelGlobalMinMax(
                         c, minMax[0], minMax[1], series);
             }
-            return null;
+            if (pixelType != FormatTools.FLOAT &&
```

Comment:
Might be easier to talk it through but what I'm seeing (in pseudo-code) is:
```
  # Old
  if (big) {
    set_defaults();
    return null;
  }
  read_planes();
```
```
  # New
  if (big) {
    set_defaults();
    if (float || double) {
      return null;
    }
  }
  read_planes();
```
where I would think we wanted:
```
  # New
  if (big) {
    set_defaults();
    return null;
  } 
  if (float || double) {
    set_defaults();
    return null;
  }
  read_planes();
```
or perhaps just
```
  # New
  if (big || float || double) {
    ...
  }
```